_includes/_article-social-share.html

```diff
@@ -4,23 +4,22 @@
 <section class="soft-half-left">
   <div class="row soft-bottom">
     <div class="flex-start flex-align-center">
-      <a href="#" role="button" class="btn btn-blue push-half-right" data-toggle="modal" data-target="#subscribeModalForm"
-      style="width: 105px;">Subscribe</a>
+      <a href="#" id="articleSubscribeButton" data-engaged="subscribed" role="button" class="btn btn-blue push-half-right" data-toggle="modal" data-target="#subscribeModalForm" style="width: 105px;">Subscribe</a>
       <div class="article-detail-social">
         <crds-heart-button theme="button" contentful-id="{{ item.contentful_id }}" contentful-slug= "{{ item.slug }}" contentful-type="{{ item.content_type | upcase }}"></crds-heart-button>
       </div>
       <div class="social-share sidebar">
-        <a href="javascript:shareFacebook()">
+        <a data-engaged="shared" href="javascript:shareFacebook()">
```

Comment:
I'm not seeing any event handlers in this PR that leverage this attribute. Is this just a handler you'll be using in GTM to trigger the event?

Reply:
@tcmacdonald There is a variable in GTM that is set off this data-engaged attribute. That variable informs a trigger that is part of the ContentEngaged tag.